nightly-master/centos/7/Dockerfile

```
@@ -0,0 +1,66 @@
+FROM centos:7
+LABEL maintainer="Swift Infrastructure <swift-infrastructure@swift.org>"
+LABEL description="Docker Container for the Swift programming language"
+
+RUN yum -y update && yum install shadow-utils.x86_64 -y \
+  binutils \
+  gcc \
+  git \
+  glibc-static \
+  libbsd-devel \
+  libedit \
+  libedit-devel \
+  libicu-devel \
+  libstdc++-static \
+  pkg-config \
+  python2 \
+  sqlite \
+  zlib-devel
+
```

```suggestion
RUN sed -i -e 's/\*__block/\*__libc_block/g' /usr/include/unistd.h

```

We need to add this, otherwise clang will fail to compile C code that includes that header (e.g. NIO fails to compile without it).